doc/make-check-bot.rst

```diff
@@ -0,0 +1,36 @@
+==============
+Make check bot
+==============
+
+The make check bot runs on every Ceph pull request and reports the
+result of ``make check`` as a comment to the corresponding pull
+request.
+
+GitLab instance
+===============
+
+A `GitLab instance <http://workbench.dachary.org/>`_ mirrors the main
+Ceph git repository and has a web hook signaling a `Jenkins instance
+<http://jenkins.ceph.dachary.org/>`_ on each pull request update.
+
+To access ``workbench.dachary.org`` which is hosted on the Red Hat
+public cloud::
+
+    ssh -A -p 2222 ubuntu@workbench.dachary.org
+
+The corresponding virtual machine is ``loic-workbench``.
+
+To access ``jenkins.ceph.dachary.org`` which is hosted by the Free
+Software Foundation France::
+
+    ssh -A ubuntu@jenkins.ceph.dachary.org
+
+Jenkins Slave
```

Comment:
This was all created in a time where we were transitioning things in our Jenkins service and build scripts, but we are standardizing in a workflow and certain conventions that makes it easy to locate scripts, build jobs, and configure them.

Instead of using something the is completely separate from the jobs that everything else related to Ceph runs, can you port this to adhere to the new conventions? It is all documented here https://github.com/ceph/ceph-build#ceph-build

Such a port would no longer require to document the make check bot this way as it would take care of all the hosting for build in an automatic way, and the Git repository is already mirrored internally.